docs/guides/getting-started/installing-cypress.mdx

```diff
@@ -214,6 +214,12 @@ on your system.
 apt-get install libgtk2.0-0 libgtk-3-0 libgbm-dev libnotify-dev libgconf-2-4 libnss3 libxss1 libasound2 libxtst6 xauth xvfb
 ```
 
+#### Arch
+
+```shell
+pacman -S gtk2 gtk2 alsa-lib xorg-server-xvfb libxss nss libnotify
```

Comment:
`gtk2` is listed twice, is one of them supposed to be something else?

Reply:
Oh sorry, it should be `gtk3`!